crates/bootstrap_srv/Cargo.toml

```diff
@@ -11,16 +11,28 @@ categories = ["network-programming"]
 edition = "2021"
 
 [dependencies]
+async-channel = { workspace = true }
+axum = { workspace = true, default-features = false, features = [
+  "http1",
+  "tokio",
+] }
 base64 = { workspace = true }
 bytes = { workspace = true }
 clap = { workspace = true, features = ["derive", "wrap_help"] }
 ctrlc = { workspace = true }
 ed25519-dalek = { workspace = true }
+futures = { workspace = true }
 num_cpus = { workspace = true }
 serde = { workspace = true }
 serde_json = { workspace = true }
 tempfile = { workspace = true }
-tiny_http = { workspace = true }
+tokio = { workspace = true, features = ["time", "rt", "rt-multi-thread"] }
```

Comment:
Actually one question - what is "time" needed for?

Reply:
[https://github.com/holochain/kitsune2/pull/52/files#diff-058df3dc5027b744df2256bd61b759aa85597ee130343b8f3f8af53cc9600034R183-R191](https://github.com/holochain/kitsune2/pull/52/files#diff-058df3dc5027b744df2256bd61b759aa85597ee130343b8f3f8af53cc9600034R183-R191)

```rust
// /crates/bootstrap_srv/src/http.rs
/*191*/     match tokio::time::timeout(std::time::Duration::from_secs(10), async move {
```